controllers/unavailability_update.py

```diff
@@ -0,0 +1,55 @@
+from flask import Blueprint, request, jsonify
+
+from flask_restful import reqparse, Resource, fields, marshal_with, Api, inputs, abort
+from domain import session_scope
+from repository.unavailability_repository import *
+
+from services.jwk import requires_auth
+
+# Request parser for creating a new unavailability event
+create_parser = reqparse.RequestParser()
+create_parser.add_argument('userId', type=int, required=True)
+create_parser.add_argument('title', type=str, required=True)
+create_parser.add_argument('periodicity', type=int, required=True)
+create_parser.add_argument('start', type=inputs.datetime_from_iso8601, required=True)
+create_parser.add_argument('end', type=inputs.datetime_from_iso8601, required=True)
+
+
+class CreateNewUnavailabilityEvent_v2(Resource):
```

Comment:
This should be named CreateUnavailabilityEventV2 (mixing title and snake case isn't good practice).